src/FastCache.Benchmarks/Comparison.cs

```diff
@@ -24,6 +25,7 @@ public class Comparison
         .WithExpiration(CacheManager.Core.ExpirationMode.Absolute, TimeSpan.FromMinutes(60)));
 
     private readonly IAppCache _lazyCache = new CachingService();
+    private readonly IFusionCache _fusionCache = new FusionCache(new FusionCacheOptions());
```

Comment:
There is no need to make it an interface (which has extra cost due to an interface call pre-.NET 8 (which has Dynamic PGO enabled by default which adds guarded devirtualization for such calls)). `IAppCache` is because that's how the original library recommends it (and the way it is written, it makes little difference regardless how you declare it here).

Reply:
Done